src/pages/_document.tsx

```diff
@@ -0,0 +1,28 @@
+import Document, { Html, Head, Main, NextScript } from "next/document";
+
+class MyDocument extends Document {
+  render() {
+    return (
```

Comment:
There is no need for a class component if the codebase is fresh (When a project is NOT a greenfield it depends on the projects convention then).

Let's go with functional components only.